Sources/Framework/Views/SingleAnswerView.swift

```diff
@@ -99,8 +94,8 @@ class SingleAnswerView: UIView {
         stackContainer.edgeAnchors == container.edgeAnchors
         horizontalStack.edgeAnchors == container.edgeAnchors + insets
 
-        answerLabel.setContentHuggingPriority(UILayoutPriorityDefaultLow, for: .horizontal)
-        iconView.setContentHuggingPriority(UILayoutPriorityRequired, for: .horizontal)
+        answerLabel.setContentHuggingPriority(UILayoutPriority.defaultLow, for: .horizontal)
+        iconView.setContentHuggingPriority(UILayoutPriority.required, for: .horizontal)
```

Comment:
It could be shorter, you don't need to specify `UILayoutPriority` struct name here:
```swift
iconView.setContentHuggingPriority(.required, for: .horizontal)
```